process-engine/bpmn-studio
# ⬆️ Update Dependencies

## Changes

1. Update Dependencies
2. Adjust BPMN Studio to PrintJS Changes

## Issues

Closes #1732

PR: #1815

## How to test the changes

- Everything should work as before except:

- Open a diagram
- Click on the print icon
- Cancel the print dialog
- **Notice that the BPMN Studio is still working.**
